### DISCUSSION TITLE
Where are databases located?

### DISCUSSION THREAD

**Discussion by e-orlov** [adminerevo/adminerevo#213] [General]
I use AdminerEvo with LocalWP on Windows10. AdminerEvo runs on `localhost:10011`. 

I fail on finding pathes to my non-Wordpress databases. Searching for `*.sql` I do find the `local.sql` database, which is those of LocalWP's Wordpress installation - but don't find my other databases I created and manage with AdminerEvo.

Could somebody point me to right pathes?

**Comment by mckaygerhard** [comment]
**adminerevo only manages already installed databases .. it does not install any database.. its just a GUI to..** adminerevo is a TCP/IP manager so databases places depends on you, services, 3thr party , etc! can be remote, local etc

hard recommendation: i suggest to you uses more systems and not like the windos so you can have a real view of the IT env
